quantcast/CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 2.1.0
+
+- Add CCPA support
```

Comment:
Hi,

We do not have CCPA test ready as of yet, only GDPR.

Can you please make a PR against canary instead of master?

Thanks.

Reply:
:+1: done